### DISCUSSION TITLE
In German transcripts, the valid word "um" is filtered out consisently

### DISCUSSION THREAD

**Discussion by dhymik** [deepgram/community#574] [General help]
### Which Deepgram product are you using?

Deepgram API

### Details

In all of my German transcripts, the word "um", which is a German word similar to "for" in english, is filtered out consistently. It is stripped out in all places.

My guess is that this is considered an unwanted utterance (in English or other languages), but in German, its a valid word and filtering it out is a bug.

We have to review the transcript and re-insert all the "ums" manually.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by pprobst** [comment]
Also having this problem! "um" in Portuguese is the number 1, so it's a serious issue IMO.
#744

**Comment by davidvonthenen** [comment]
Hi @pprobst and @dhymik 

If you want all those "conversation pause"-type words, you need to enable filler words:
https://developers.deepgram.com/docs/filler-words

**Comment by pprobst** [comment]
This is not "solved"; this is a significant bug in German and Portuguese since "um" is a word. In Portuguese, it's both a very common indefinite article and the number one. So *a lot* of transcriptions will be broken.

**Comment by davidvonthenen** [comment]
Were you able to try and see if it worked for you?

Are you using nova-2 for the model?

If that doesn't work, a hacky way to increase the prominence of a word is to use keywords:
https://developers.deepgram.com/docs/keywords

**Reply by dhymik** [reply]
I agree with @pprobst that enabling filler words is really not a solution, because this will negatively affect the quality of the transcript.

Yes, I am using nova-2.

I have aded "um" as a keyword for transcripts of German audio and will get back to you here as soon as I have results.

**Reply by pprobst** [reply]
> Were you able to try and see if it worked for you?
> 
> Are you using nova-2 for the model?
> 
> If that doesn't work, a hacky way to increase the prominence of a word is to use keywords: [developers.deepgram.com/docs/keywords](https://developers.deepgram.com/docs/keywords)

Sorry, I forgot the topic. I did a test with filler words a couple of weeks ago and it successfully recognized "um". However, unwanted filler words inevitably occur during the transcription depending on the speaker. I have not tried hotwords; for now, I'm not using Deepgram anymore.

**Reply by dhymik** [reply]
@dvonthenen I tested today with adding "um" to the keywords but this made no difference. "Um" still missing in the transcripts.

The only effect is that other words, which are not "um", are now occasionally recognized as "um". But the actual "um"s are still missing.

What can be done?

**Reply by davidvonthenen** [reply]
hey @dhymik and @pprobst 

Thanks for giving that a try. Workarounds have worked like this in the past (it seems to have worked for one of you but not the other), but apparently not this time. I will pass this along to product and engineering to see if they can prioritize this issue.

@pprobst if you were able to get it to recognize "um" but are now getting the other filler words, you can use the replace function to remove the other filler words to an empty string. This isn't ideal, but it's what's available. Otherwise, regardless, passing this along for a proper fix.

**Comment by madmus2025** [comment]
The problem is annoying in Nova 2, Nova 3 and even via Whisper/OpenenAI in Deepgram - we have therefore deactivated Deepgram as a transcription provider.

**Comment by dhymik** [comment]
Any news on this almost 2 year old issue? I have to edit every single German transcript by hand because of this word being filtered out.

To give you a perspecitve of the scope of this problem: it is as if you would filter out the english word "for"...
